test/build/bugfix-named-import/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Tests for the _$v4 named import bug",
+  "snowpack": {
```

Comment:
Do you think it would be useful to add a link to the ticket?

Reply:
Ah good idea. Added